client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml

```diff
@@ -303,6 +303,10 @@
    <translation id="privateDatabase_add_bdd_new_user_password_confirm" qtlid="300539">Confirmer le mot de passe</translation>
    <translation id="privateDatabase_add_bdd_new_user_password_tooltip" qtlid="300552">Entre 8 et 31 caractères, au moins une lettre majuscule, une lettre minuscule et un chiffre, doit être composé uniquement de chiffres et de lettres</translation>
 
+   <translation id="privateDatabase_add_authorized_ip">Ajouter une adresse IP autorisée</translation>
+   <translation id="privateDatabase_add_authorized_ip_explanation">Vous n'avez pas encore d'adresse IP autorisée à accéder à vos bases de données. Entrez ici l'adresse IP du serveur auquel vous souhaitez donner accès à vos bases.</translation>
```

Comment:
This sentence doesn't feel right, wouldn't `Aucune adresse IP n'est autorisée à accéder à vos bases de données. Vous pouvez insérer ici l'adresse IP du ou des serveur(s) qui y auront accès` or something along this way be better ?

(I changed in case more than one server have access to the databases).